.github/workflows/uberjar.yml

```diff
@@ -0,0 +1,77 @@
+name: Uberjar
+
+on:
+  push:
+    branches:
+      - master
+      - 'release-**'
+    paths-ignore:
+    - 'docs/**'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    strategy:
+      matrix:
+        edition: [ee, oss]
+    env:
+      MB_EDITION: ${{ matrix.edition }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Prepare Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+    - name: Prepare JDK 8
```

Comment:
I think we're running Java 11 in CircleCi. Unless we're constrained by anything and have to use v8 here, wouldn't it be better to match 2 environments more closely?

Reply:
Good catch!

Reply:
we're running tests with both Java 8 and Java 11 on CircleCI. Build currently only works on Java 8 due to a bug with GNU gettext tho.

Reply:
Thanks @camsaul. I confirmed that by checking the container image `circleci/clojure:lein-2.8.1-node-browsers`:

```
$ docker run -it circleci/clojure:lein-2.8.1-node-browsers java -version
openjdk version "1.8.0_181"
OpenJDK Runtime Environment (build 1.8.0_181-8u181-b13-2~deb9u1-b13)
OpenJDK 64-Bit Server VM (build 25.181-b13, mixed mode)
```